core/src/main/kotlin/xtdb/indexer/LiveTable.kt

```diff
@@ -31,6 +31,7 @@ class LiveTable
     private val trieWriter = TrieWriter(al, bp, false)
     private val trieMetadataBuilder = TrieMetadata.newBuilder()
     private var iidBloom = RoaringBitmap()
+    private val hyperLogLogs = mutableMapOf<String, HLL> ()
```

Comment:
I'd be fine with calling these `hlls`, I think, save both your fingers and my eyes :sweat_smile: